pkg/virtctl/imageupload/imageupload.go

```diff
@@ -411,3 +419,17 @@ func getAndValidateUploadPVC(client kubernetes.Interface, namespace, name string
 
 	return pvc, nil
 }
+
+func getUploadProxyURL(client cdiClientset.Interface) (string, error) {
+	cdiConfig, err := client.CdiV1alpha1().CDIConfigs().Get("config", metav1.GetOptions{})
```

Comment:
Does the CDI installation take care of RBAC? Is this accessible via RBAC aggregation? Does it only contain non-sensitive information by definion or is it just an "admin" helper?

Reply:
This is just an admin helper, non-sensitive information. CDI takes care of RBAC.